### PR TITLE
Add point cloud demo

### PR DESCRIPTION
**Pull request by SeptimiuVana**

### CODE REVIEW

File: doc/linux/build_instructions.md [hunk 1]
```diff
@@ -77,3 +77,32 @@ make
...
+ The steps required for building and including the required dependencies are presented below:
+ 
+ Requirements:
+ * OpenCV 4.5.0
+
+```console
+sudo apt-get install libgtk-3-dev libvtk6-dev
+git clone --branch 4.5.0 --depth 1 https://github.com/opencv/opencv.git
+git clone --branch 4.5.0 --depth 1 https://github.com/opencv/opencv_contrib.git
+cp -r .\opencv_contrib\modules\viz .\opencv\modules\
```
Comment (rbudai98) [inline doc/linux/build_instructions.md:93]: backslash problem in my case:
cp: cannot stat '.opencv_contribmodulesviz': No such file or directory
cp: cannot stat '.opencvmodulescd': No such file or directory

File: doc/linux/build_instructions.md [hunk 2]
```diff
@@ -77,3 +77,32 @@ make
...
+sudo cmake --build . --target install
+```
+
+Compile the sdk and add OpenCV to CMAKE_PREFIX_PATH:
+
+```console
+cd aditof_sdk
+mkdir build
+cd build
+cmake -DCMAKE_PREFIX_PATH="/opt/glog;/opt/protobuf;/opt/websockets;/opt/opencv"" -DWITH_EXAMPLES=on ..
```
Comment (rbudai98) [inline doc/linux/build_instructions.md:106]: Double quotation marks

File: doc/linux/build_instructions.md (hunk 1 above)
Comment (SeptimiuVana) [reply doc/linux/build_instructions.md]: Fixed, thank you!

File: doc/linux/build_instructions.md (hunk 2 above)
Comment (SeptimiuVana) [reply doc/linux/build_instructions.md]: Fixed.

File: doc/linux/build_instructions.md [hunk 3]
```diff
@@ -77,3 +77,32 @@ make
 ## SDK with bindings
 
 - Please check the readme files for each type of binding in the [bindings directory](../../bindings).
+
+### Enabling the point cloud display in aditof-demo
+
+ OpenCV needs to be build manually in order to visualize pointCloud data because the prebuild libraries do not come with the viz module installed.
```
Comment (acozma) [inline doc/linux/build_instructions.md:83]: *built

File: doc/windows/build_instructions.md [hunk 4]
```diff
@@ -131,4 +131,44 @@ Use parameter *-h* or *--help* for more information regarding the parameters.
 - Run the application.
 ![Display Image](/doc/img/run_VS.PNG) 
 
-- Application executable is created in **current_script_path\build\example\aditof-demo\Release** directory.
\ No newline at end of file
+- Application executable is created in **current_script_path\build\example\aditof-demo\Release** directory.
+
+### Enabling the point cloud display in aditof-demo
+
+ OpenCV needs to be build manually in order to visualize pointCloud data because the prebuild libraries do not come with the viz module installed.
```
Comment (acozma) [inline doc/windows/build_instructions.md:138]: *built

**Review by acozma — CHANGES_REQUESTED** [review]
Looks good, just need to fix the spelling in the instructions before merging

File: doc/linux/build_instructions.md (hunk 3 above)
Comment (SeptimiuVana) [reply doc/linux/build_instructions.md]: Fixed, thank you!

File: doc/windows/build_instructions.md (hunk 4 above)
Comment (SeptimiuVana) [reply doc/windows/build_instructions.md]: Fixed.

**Review by dNechita — APPROVED** [review]

**Review by acozma — APPROVED** [review]

**Pull request merged**